tests/reference/cpp-doconcurrentloop_01-4e9f274.stdout

```diff
@@ -41,8 +41,8 @@ void triad(const Kokkos::View<float*> &a, const Kokkos::View<float*> &b, float s
 {
     int N;
     N = 1234;
-    Kokkos::parallel_for(Kokkos::RangePolicy<Kokkos::DefaultExecutionSpace>(1, N - 1+1), KOKKOS_LAMBDA(const long i) {
-        c[i-1] = a[i-1] + scalar*b[i-1];
+    Kokkos::parallel_for(Kokkos::RangePolicy<Kokkos::DefaultExecutionSpace>(1, N - (1)+1), KOKKOS_LAMBDA(const long i) {
+        c[i-1] = a[i-1] + (scalar) * (b[i-1]);
```

Comment:
This change seems wrong for two reasons:
* there should be no parentheses
* There shouldn't be space around `*`. This is to follow SymPy's approach, of putting space around + and -, but not * and /. It seems the expression is generally more readable that way.